src/block-components/alignment/edit.js

```diff
@@ -346,6 +348,18 @@ export const Edit = props => {
 	)
 }
 
+// Add additional classes when browser is Firefox to fix alignment
+addFilter( 'stackable.block-components.block-div.classnames', 'alignment-editor-has-polyfill', classes => {
+	const userAgent = navigator.userAgent
+	if ( userAgent.indexOf( 'Firefox' ) !== -1 ) {
+		const alignmentHasPolyfill = EditorHasPolyfill()
+		const classesToAdd = Object.keys( alignmentHasPolyfill ).filter( classname => alignmentHasPolyfill[ classname ] )
+		const newClasses = classes.concat( classesToAdd )
+		return newClasses
```

Comment:
If you want to add classes returned by this filter, you can just append things in the argument array and then return that:

```js
return [
    ...classes,
    ...newClasses,
]
```

or 

```js
classes.push( newClass )
return classes
```